irr/src/OpenGL/Driver.cpp

```diff
@@ -107,6 +107,12 @@ static const VertexType &getVertexTypeDescription(E_VERTEX_TYPE type)
 		return vtTangents;
 	default:
 		assert(false);
+		std::abort();
+#ifdef __GNUC__
+		__builtin_unreachable();
```

Comment:
we don't need both this and `abort`

it would also make sense to define this as macro somewhere so it can be reused.

Reply:
Fixed in 9f326b529. I don't know if `irr/src/os.h` is the correct place to add it, so please review.